pkg/apis/deviceplugin/v1/qatdeviceplugin_webhook.go

```diff
@@ -91,5 +91,9 @@ func (r *QatDevicePlugin) validatePlugin() error {
 		}
 	}
 
+	if len(r.Spec.ProvisioningConfig) > 0 && len(r.Spec.InitImage) == 0 {
```

Comment:
`ProvisioningConfig` is only valid if `4xxxvf` is enabled so maybe we could check that here too?

Reply:
I added, but suddenly am not sure actually how vfKernelDrivers work when there are many different ones.